app/models/concerns/findings/reiterations_oracle.rb

```diff
@@ -0,0 +1,90 @@
+module Findings::ReiterationsOracle
```

Comment:
```suggestion
module Findings::ReiterationsAlt
```
Ya que para SQLite va a usarse ese también... mientras dure esa rama.

Reply:
Arrepientente pecador!